site/diagrams/sequenceDiagrams/command_control_concept_http.txt

```diff
@@ -0,0 +1,18 @@
+participant Device
+participant HTTP Adapter
+participant AMQP 1.0 Network
+participant Application
+
+Device->HTTP Adapter: 1. send event (hono-ttd=30)
+activate HTTP Adapter
+HTTP Adapter->Application: 2. connected notification (ttd=30)
+activate Application
+Application->HTTP Adapter: 3. transfer("control/TENANT/4711","light","ON")
+HTTP Adapter->Device: 4. event response("light","ON",${reqId})
+deactivate HTTP Adapter
+note over Device
+Command response in next operation
+end note over
+Device->HTTP Adapter: 5. /control/res("hono-cmd=light","ON",${reqId})
+HTTP Adapter->Application: 6. transfer("control/TENANT/4711/${replyId}",200,"lumen=100")
```

Comment:
`${replyTo}`?
Where does the `200` come from? If this is the result status of executing the command on the device then it needs to be passed in from the device in its request, e.g.
`HTTP request("POST", "/control/res/${reqId}/200", "lumen=100")`

Reply:
Yes, I forgot the status in the diagram - thanks!